Dockerfile.local

```diff
@@ -1,32 +1,12 @@
-FROM ubuntu:xenial
+FROM node:6
 MAINTAINER LiskHQ
 LABEL description="Lisk Docker Image - Local Testnet" version="1.4.2"
```

Comment:
I think we need a new version in here. I'd say `1.5.0` or directly `2.0.0`. How do you fell @Isabello ?

Reply:
I will bump version to 2.0.0 after this merge and release it as such